app/javascript/src/components/Dashboard/Notes/NotesCard.jsx

```diff
@@ -29,13 +23,21 @@ const NotesCard = ({ note, deleteNote }) => {
           </Dropdown>
         </div>
         <div className="text-gray pt-2 px-4">
-          <p className="text-sm pr-1">{note.content}</p>
+          <p className="text-sm pr-1">{note.description}</p>
```

Comment:
Can't we use `Typography` component here?

Reply:
Yes will do that 👍